doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside_boot.rst

```diff
@@ -0,0 +1,436 @@
+
+
+
+
+.. _ug_nrf54h20_ironside_se_boot_report:
+
+|ISE| boot
```

Comment:
```suggestion
.. _ug_nrf54h20_ironside_boot:
.. _ug_nrf54h20_ironside_se_booting:

Boot management
```

"ISE Boot" sounds specific to our boot, but this page contains info on various things around booting and reset handling for both ISE and local domains.